hack/arktos-up-scale-out-poc.sh

```diff
@@ -681,6 +687,11 @@ if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
 fi
 
 if [ "${IS_RESOURCE_PARTITION}" != "true" ]; then
+  # Applying mizar cni
+  if [[ "${CNIPLUGIN}" == "mizar" ]]; then
+    # ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f ${KUBE_ROOT}/hack/testdata/mizar/deploy.mizar.next.yaml 
+    ${KUBECTL} --kubeconfig="${CERT_DIR}/admin.kubeconfig" apply -f https://raw.githubusercontent.com/CentaurusInfra/mizar/dev-next/etc/deploy/deploy.mizar.dev.yaml
```

Comment:
Missing latest change to mizar yaml file

Reply:
Original idea is after merge this PR, send another PR to contain latest changes.  today I will publish another large PR to contain 4 commits of local script changes after I complete the tests for mizar at least. Thanks!